addons/website/static/src/snippets/s_media_list/001.scss

```diff
@@ -0,0 +1,13 @@
+.s_media_list[data-vcss="001"] {
```

Comment:
There was `.s_media_list_options` in the old scss, was this used at all?

Reply:
Last seen in saas-13.1.

Reply:
:+1: